cfme/common/provider.py

```diff
@@ -192,7 +233,8 @@ def create(self, cancel=False, validate_credentials=True, check_existing=False,
                 if cancel:
                     created = False
                     add_view.cancel.click()
-                    cancel_text = 'Add of Infrastructure Provider was cancelled by the user'
+                    cancel_text = 'Add of {} Provider was ' \
+                                  'cancelled by the user'.format(self.string_name)
```

Comment:
@quarckster let's discuss this together and decide which style we are going to use.